### ISSUE TITLE
Jinja2 templates will spam network requests.

### ISSUE THREAD

**Issue by bratanon** [MizterB/lovelace-secondaryinfo-entity-row#47]
When using the jinja2 template, this card will request the template endpoint multiple times every second. 

```
  - entity: light.entrance_outdoor    
    type: "custom:secondaryinfo-entity-row"
    secondary_info: "{{ 1 + 1 }}"
```

![image](https://user-images.githubusercontent.com/812265/146159479-77c8a5ab-e012-4267-be1b-3cb643d57069.png)

Guess this is because `_updateElement` will trigger as soon as something changes in the `hass` object, which for a larger setup could be many hundred times per second.

**Comment by igorsantos07** [comment]
Yep, I noticed the same while looking around for a way to use `{entity}` inside the Jinja templates. Gave up on opening a new issue, as we'd better get back to the weird `[[ ]]` syntax, given this seems unavoidable for being present since 2021.

>Edit: or maybe not, bdraco seems to have made a suggestion that could fix this issue at #53.
